.github/workflows/release.yml

```diff
@@ -0,0 +1,115 @@
+name: build-images
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+env:
+  GHCR_IMAGE: ghcr.io/cynova/directus
```

Comment:
We can't have it build from an out of org image

Reply:
sorry, my bad this was left there from testing. i'll fix it right now